blog/config/common/di/router.php

```diff
@@ -11,38 +11,35 @@
 use Yiisoft\Router\RouteCollectionInterface;
 use Yiisoft\Router\RouteCollectorInterface;
 use Yiisoft\Router\UrlGeneratorInterface;
-use Yiisoft\Yii\Debug\Viewer\Middleware\ToolbarMiddleware;
 
 /**
  * @var Config $config
  * @var array $params
+ * @var array $params['yiisoft/router-fastroute']
+ * @var bool $params['yiisoft/router-fastroute']['encodeRaw']
+ * @var array $defaultArguments 
+ * @psalm-suppress MixedArgument $routes
  */
 
 return [
     UrlGeneratorInterface::class => [
         'class' => UrlGenerator::class,
         'setEncodeRaw()' => [$params['yiisoft/router-fastroute']['encodeRaw']],
         'setDefaultArgument()' => ['_language', 'en'],
-        'reset' => function () {
-            $this->defaultArguments = ['_language', 'en'];
+        'reset' => function (array $defaultArguments = []) {
+            $defaultArguments = ['_language', 'en'];
         },
     ],
-
+    
     RouteCollectionInterface::class => static function (RouteCollectorInterface $collector) use ($config) {
+        $routes = $config->get('routes');
         $collector
             ->middleware(CsrfMiddleware::class)
             ->middleware(FormatDataResponse::class)
-            ->addGroup(
-                Group::create('/{_language}')->routes(...$config->get('app-routes')),
-            )
-            ->addGroup(
-                Group::create()->routes(...$config->get('routes')),
```

Comment:
Was that created to exclude debug routes from language routing?

Reply:
key  'app-routes' exists in the configuration.php file whereas  key 'routes' does not. Just thought routes was a better naming convention for the  ['common/routes/*.php']. My configuration file key 'routes' in rossaddison/invoice {root}configuration.php points to [
            'common/routes/*.php' as well but I do not have an app-routes key instead I have 'routes'
        ]. So the package still picks up all php files in the routes folder for merging. So suggest condensing the two into one.